documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,575 @@
+process are as follows:
+
+- Rather than tying the question and reaction rounds together, they are separate
+  rounds. Therefore, the rounds are as follows:
+  1. Question
+  1. Reaction
+  1. Revision
+  1. Objection
+  1. Objection revision
+- Each round should have notes taken clearly documenting the questions,
+  reactions, objections, and discussions for each (as appropriate). Priority
+  should be given to questions and objections that require revising the
+  proposal.
+
+## How to follow the process in different settings
+
+The team has agreed to use GitHub for all discussions, with links shared on the
+[WordPress Make Openverse blog](https://make.wordpress.org/openverse/) to help
+visibility for certain discussions.
+
+There are two tools in GitHub that can be used for conducting a discussion: PRs
+and GitHub Discussions. We do not use GitHub issues because they do not allow
+for threading.
+
+### PRs
+
+GitHub PRs are used for project proposals, implementation planning RFCs, and
+other proposals that will be documented as Markdown files in the repository.
+
+GitHub PRs have the following ways of interacting with comments:
+
+- Inline comments on specific lines of text or code
+  - These allow opening "threads" and can be resolved
+- General PR comments
+- Review comments
+
+Each of these can serve their own purposes:
+
+- Inline comments
+  - These are perfect to supplement the question and objection rounds. If a
+    question is about a specific part of the proposal (rather than a more
+    general clarification), participants can leave the question directly on the
+    part in question. Similarly, paramount objections specific to one part of
+    the proposal can be left as inline comments attached to a review requesting
+    changes.
+- General PR comments
+  - For questions or non-paramount objections raised regarding the proposal in
+    general or for the reaction round.
+  - In addition to the PR description, the author of the PR will also create a
+    new top-level PR comment when a new round starts.
+- Review comments
+  - These should be used in the objection raising round. Leave an approving
+    review if you have no paramount objections or "request changes" if you have
+    a paramount objection. Attach inline comments as needed in either case.
+
+### GitHub Discussions
+
+Because the format is slightly more limited than PRs, we will have a top-level
+comment per round created by the author. Responses from participants should be
+contained within the thread for each round. However, paramount objections during
+the Objection round, should be raised as individual threads. Still, the author
+should still create a top-level comment closing the previous round and opening
+the objection round. If the proposal goes into objection revision, the
+discussion for addressing a given objection can happen in the accompanying
+thread.
+
+## Major differences from the original process
```

Comment:
I'm not sure whether to pre-emptively resolve this so will leave it up. The part about potentially moving this into a separate document is definitely 'formatting' work which, to reiterate, we definitely don't need to address now.

Regardless of where the documentation lives, I think the point about it being difficult to explain the differences without explaining the original method stands. I'm leaving this as a _suggestion_ that removing it (temporarily, if folks think it would be valuable to add back in) and not trying to directly compare our process to the quite-different inspiration would be the minimal effort approach for a v1 doc.